docs/index.asciidoc

```diff
@@ -343,10 +343,11 @@ Timeout (in seconds) for the entire request
   * Value type is <<boolean,boolean>>
   * Default value is `true`
 
-Note that this option controls retries for plugin-level requests only. 
+Note that this option controls plugin-level retries only. 
```

Comment:
My change was misleading. We don't have plugin-level requests, instead, we have plugin-level retries